Description: fix the sfxload.1 manpage
 Fix several typos in sfxload.1 manpage.
Author: Noel Koethe A. Costa <agcosta@gis.net>
Forwarded: no
Last-Update: 2007-08-07

Index: awesfx-0.5.1e/sfxload.1
===================================================================
--- awesfx-0.5.1e.orig/sfxload.1
+++ awesfx-0.5.1e/sfxload.1
@@ -1,5 +1,4 @@
 .TH sfxload 1 "January 22, 2003"
-.LO 1
 .SH NAME
 sfxload, asfxload \- load a SoundFont file on the Emux WaveTable
 
@@ -20,7 +19,7 @@ or to the Emux WaveTable of ALSA sbawe a
 This program is necessary for playing MIDI samples via sequencer
 programs supporting AWE driver.  There is no big difference between
 \fBsfxload\fP and \fBasfxload\fP except for that \fBasfxload\fP is for
-ALSA and \fBsfxload\fP is for OSS, respecitvely. The options to specify
+ALSA and \fBsfxload\fP is for OSS, respectively. The options to specify
 devices are
 different between them (see below).
 
@@ -70,7 +69,7 @@ Either of \fBon\fP, \fBoff\fP, \fByes\fP
 or \fBfalse\fP can be specified as an optional argument.
 .TP
 .BI \-x,\ \-\-remove "\fR[=\fPbool\fR]\fP"
-Remove the optional samples previouly loaded via \fB\-b\fP option.
+Remove the optional samples previously loaded via \fB\-b\fP option.
 Otherwise, all new samples are simply appended.
 .TP	
 .BI \-N,\ \-\-increment "\fR[=\fPbool\fR]\fP"
@@ -161,7 +160,7 @@ The preset is given as same as in virtua
 Increase or set the verbosity level.
 .TP
 .BI \-q,\ \-\-quiet
-Don't show error messages, equivalen with \-\-verbose=0.
+Don't show error messages, equivalent with \-\-verbose=0.
 .TP
 .BI \-V,\ \-\-volume= percent
 Specify the total volume of sounds, provided in percent.
@@ -275,7 +274,7 @@ at first.
 .SH COPYRIGHT
 Copyright (C) 1996-2003 Takashi Iwai.
 .P
-The AWE32 driver and utilties are free software; you can redistribute them
+The AWE32 driver and utilities are free software; you can redistribute them
 and/or modify them under the terms of the \fIGNU General Public
 License\fP as published by the Free Software Foundation; either
 version 2 of the License, or (at your option) any later version.
